### ISSUE TITLE
output vector is all zeros of project_gaussians

### ISSUE THREAD

**Issue by lixinhaiyk** [LightwheelAI/street-gaussians-ns#26]
These input values are all correct, but the output vector is all zeros.Has anyone encountered this issue before?
the version is gsplat==0.1.2.1

```
        self.xys, self.depths, self.radii, self.conics, self.num_tiles_hit, self.cov3d__ = project_gaussians(  # type: ignore
            means_crop,
            scales_crop,
            1,
            quats_crop / quats_crop.norm(dim=-1, keepdim=True),
            viewmat.squeeze()[:3, :],
            projmat,
            camera.fx.item(),
            camera.fy.item(),
            cx,
            cy,
            H,
            W,
            #self.config.block_width,
            tile_bounds,
        )  # type: ignore
```

**Comment by ChengenXie** [comment]
It seems like the original code is
```
self.xys, self.depths, self.radii, self.conics, _, self.num_tiles_hit, _ = project_gaussians(  # type: ignore
            means_crop,
            scales_crop,
            1,
            quats_crop / quats_crop.norm(dim=-1, keepdim=True),
            viewmat.squeeze()[:3, :],
            camera.fx.item(),
            camera.fy.item(),
            cx,
            cy,
            H,
            W,
            self.config.block_width,
        )  # type: ignore
```
And I found there misses a parameter projmat, how to calculate it?

**Comment by lixinhaiyk** [comment]
"Yes, but the interface for gsplat==0.1.2.1 version is like this. I had to
adjust it to get it running."

Bob Xie ***@***.***> 于2024年7月4日周四 16:31写道：

**Comment by lixinhaiyk** [comment]
Yes, but the interface for gsplat==0.1.2.1 version is like this. I had to adjust it to get it running.

**Comment by Lazyangel** [comment]
> Yes, but the interface for gsplat==0.1.2.1 version is like this. I had to adjust it to get it running.

I just run  `pip install gsplat==0.1.11` and the problem seems to be solved，you can give it a try.

**Comment by mzlchou** [comment]
> Yes, but the interface for gsplat==0.1.2.1 version is like this. I had to adjust it to get it running.

What did you define tile_bounds as?

**Comment by mzlchou** [comment]
> pip install gsplat==0.1.11

It says that "nerfstudio 1.0.0 requires gsplat==0.1.2.1, but you have gsplat 0.1.11 which is incompatible." How did you get around this?

**Comment by Lazyangel** [comment]
> > pip install gsplat==0.1.11
> 
> It says that "nerfstudio 1.0.0 requires gsplat==0.1.2.1, but you have gsplat 0.1.11 which is incompatible." How did you get around this?
install `nerfstudio=1.0.3`and`gsplat==0.1.11`, and no need to modify the code after installation.

**Comment by mzlchou** [comment]
> nerfstudio=1.0.3

Thank you, not sure if you would know but I am now getting this error: "TypeError: project_gaussians() missing 1 required positional argument: 'block_width'" Did you run into this?

**Comment by mzlchou** [comment]
> > > pip install gsplat==0.1.11
> > 
> > 
> > It says that "nerfstudio 1.0.0 requires gsplat==0.1.2.1, but you have gsplat 0.1.11 which is incompatible." How did you get around this?
> 
> install `nerfstudio=1.0.3`and`gsplat==0.1.11`, and no need to modify the code after installation.

Sorry to keep bothering you, but I got this error "ImportError: DLL load failed while importing gsplat_cuda: The specified module could not bund." Did you get anything like this?

**Comment by Lazyangel** [comment]
> > > > pip install gsplat==0.1.11
> > > 
> > > 
> > > It says that "nerfstudio 1.0.0 requires gsplat==0.1.2.1, but you have gsplat 0.1.11 which is incompatible." How did you get around this?
> > 
> > 
> > install `nerfstudio=1.0.3`and`gsplat==0.1.11`, and no need to modify the code after installation.
> 
> Sorry to keep bothering you, but I got this error "ImportError: DLL load failed while importing gsplat_cuda: The specified module could not bund." Did you get anything like this?

Sorry, I haven't encountered similar problems
